docs/custom_tools.md

```diff
@@ -36,3 +40,31 @@ version of helm than what is bundled in Argo CD:
           name: custom-tools
           subPath: helm
 ```
+
+## BYOI (build your own image)
```

Comment:
No because in my example, I didn't have to build my container-- I got it off the shelf. I will change the example to use something like alpine so it's obvious the advantage of that approach..